lms/djangoapps/certificates/tasks.py

```diff
@@ -18,11 +19,29 @@ class _BaseCertificateTask(PersistOnFailureTask, LoggedTask):  # pylint: disable
     abstract = True
 
 
-@task(base=_BaseCertificateTask)
-def generate_certificate(**kwargs):
+@task(base=_BaseCertificateTask, bind=True, default_retry_delay=30, max_retries=2)
+def generate_certificate(self, **kwargs):
     """
     Generates a certificate for a single user.
+
+    kwargs:
+        - student: The student for whom to generate a certificate.
+        - course_key: The course key for the course that the student is
+            receiving a certificate in.
+        - expected_verification_status: Either 'approved' or None. Passed in
```

Comment:
:) If this is present in `kwargs`, it won't be None, based on the change you made yesterday.